### DISCUSSION TITLE
팀원 수다방

### DISCUSSION THREAD

**Discussion by pkwon35** [pkwon35/Smart_Factory_APS_System#1] [General]
하고싶은말 막 하는방~~~~

**Comment by pkwon35** [comment]
새벽 2시... 졸리다아아아아

**Comment by pkwon35** [comment]
10월25일 선생님이랑 멘토링 성공!! 이제 코딩만 빠박

**Comment by pkwon35** [comment]
선생님이 더 많은 데이터를 주셨고.... 우리는 더 많은 고민을 해야한다!!!!

**Comment by kksonge** [comment]
어느 시점의 날씨, 판매량, 요일이 상관계수가 높은지 확인하기

**Comment by pkwon35** [comment]
![20211029_160537.jpg](https://user-images.githubusercontent.com/25214976/139431775-352ec555-4f7c-4532-b740-53561a080e68.jpg)

팀플은 캠핑장에서

**Reply by pkwon35** [reply]
구독과 좋아요~~~ 부탁함다  feat 옥상게스트하우스 사장

**Comment by pkwon35** [comment]
으어어어어어어어어어어어어ㅓㅓㅇ  실행파일로 안만들어져요........................ㅜㅜㅜㅜㅜㅜㅜㅜㅠㅠㅠㅠㅠㅠㅠ

**Comment by pkwon35** [comment]
모닝콜... 부탁해여....ㅠ

**Comment by pkwon35** [comment]
살려줘염너이ㅏ러ㅣㅠㅠ

**Comment by JINYUHOON** [comment]
![eee](https://user-images.githubusercontent.com/68021998/139703386-73b4f0e2-cb6b-4360-95a5-4337cc9e29e4.jpg)
